src/vllm_tgis_adapter/grpc/grpc_server.py

```diff
@@ -194,7 +194,7 @@ async def post_init(self) -> None:
         assert hasattr(self.engine.engine, "stat_logger")
```

Comment:
So while testing this out, I missed making changes to this line, and interestingly, the server did NOT crash while booting up, instead it failed when I tried to close it - 
```
INFO:     Shutting down
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.
[rank0]: Traceback (most recent call last):
[rank0]:   File "<frozen runpy>", line 198, in _run_module_as_main
[rank0]:   File "<frozen runpy>", line 88, in _run_code
[rank0]:   File "/home/develop/.conda/envs/vllm/lib/python3.11/site-packages/vllm_tgis_adapter/__main__.py", line 311, in <module>
[rank0]:     event_loop.run_until_complete(grpc_server_task)
[rank0]:   File "/home/develop/.conda/envs/vllm/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
[rank0]:     return future.result()
[rank0]:            ^^^^^^^^^^^^^^^
[rank0]:   File "/home/develop/.conda/envs/vllm/lib/python3.11/site-packages/vllm_tgis_adapter/grpc/grpc_server.py", line 930, in run_grpc_server
[rank0]:     server = await start_grpc_server(engine, args)
[rank0]:              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/develop/.conda/envs/vllm/lib/python3.11/site-packages/vllm_tgis_adapter/grpc/grpc_server.py", line 859, in start_grpc_server
[rank0]:     await generation.post_init()
[rank0]:   File "/home/develop/.conda/envs/vllm/lib/python3.11/site-packages/vllm_tgis_adapter/grpc/grpc_server.py", line 198, in post_init
[rank0]:     assert hasattr(self.engine.engine, "stat_logger")
[rank0]:            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[rank0]: AssertionError
```

That seems like a bug?